### ISSUE TITLE
Port ADR to IM151

### ISSUE THREAD

**Issue by vendethiel** [IntegraMOD/IntegraMOD151#52]
I suppose it should be as a branch.

Not having to maintain AD&R on EzArena as well as on IM would be helpful...

**Comment by vendethiel** [comment]
@MWE001 How do you think we should do the ACP?
I can just setup all the categories, but that bloats the ACP an incredible amount, and the ADR categories get at the top.
Two ways I can try to go about it:

 - On EzArena, there's a switch between board admin and ADR admin
![image](https://user-images.githubusercontent.com/199499/47559277-5f3a5880-d915-11e8-9bf0-79e1223578e2.png)

 - ADR being a submenu? But that means you have a menu in a menu... (ADR > General > Configuration).


What do you think? Suggestions?

**Comment by MWE001** [comment]
I would think The admin panel switch between the two would be decent. ADR is quite "Fluffy" in the admin panel and goes on and on. I think if it had its own ACP so to speak that would be much better.

**Comment by vendethiel** [comment]
This is implemented on ADR branch. Really I just need to take a look at PCP now. But I have been busy with bug fixes for now.

**Comment by IntegraMOD** [comment]
Check ACP/Photo Album.
Especially check the configuration link. It might be a good example

**Comment by vendethiel** [comment]
Just finished writing a install.php file for AD&R's SQL procedure.

**Comment by vendethiel** [comment]
adr_character.php is working all fine!
I fixed the SQL to run on recent-ish servers.

**Comment by vendethiel** [comment]
OK, now that PCP is integrated, I wonder what's the next step.
Should I make AD&R a module that can be just drag'n'dropped?

This means leaving some AD&R code here and there, disabled unless some file exists.

This is a way to do things that was proposed on the Integramod forums a few years back.
Maybe I can make a system generic enough, but honestly... we only have AD&R for now.

This is also a way for me not to have to maintain 2 integramod branches...

**Comment by MWE001** [comment]
Ever come up with anything on this Chief?

**Comment by vendethiel** [comment]
Well, AD&R was integrated with PCP. It doesn't take advantage of PCP in its entirety, mostly because no one here wants to configure PCP I think.

I'm still waiting for people's opinion on this. Obviously, shipping Integramod with AD&R for everyone is way too much.

**Comment by MWE001** [comment]
I agree. That would be way too much for 99% of the people that might still want to use the system. PCP is a PITA to configure. All of the PCP masters we had at Integra are way since long gone. I dabbled with it for a while and got a few mods to work with it and was able to configure it to work with a few others just by tweaking some settings via the ACP but man alive was it horrible. I hated every minute of it.

**Comment by vendethiel** [comment]
I also tend to mess up, press the buttons twice, and then the whole site is blank until I restore the file from GIT. I dont know if youve had that... It uses files instead of the database for no good reason I think. Anyway 2020 is not the year to spend doing PCP stuff. Only interesting stuff.
